src/main/fulcro/client.cljc

```diff
@@ -164,6 +164,16 @@
                                              reconciler-options)
                        :networking         networking})))
 
+
+(defn new-fulcro-client
+  "Deprecated
+  "
+  [& {:as options}]                                         ; hack ?  [& options] returned IndexedSeq
+  (let [options-with-marker-default (update options :reconciler-options assoc :inplace-load-markers true)
```

Comment:
perhaps should call it "load-marker-default", since we could let them set the default to a keyword or something as well.